js/MVP/views/landing/Landing.tsx

```diff
@@ -61,10 +61,10 @@ const Landing = (): ReactElement => {
   };
 
   const displayBrev = (): boolean => {
-    if (brev.isError || brev.data.length === 0) {
+    if (brev.isIdle || brev.isError || !brev.data || brev.data.length === 0) {
```

Comment:
Hm. Kan denne forenkles til enten `!brev.isSuccess || brev.data.length === 0` eller alternativt enda kortere: `brev.data?.length === 0`

Reply:
Ja, på noen av disse sjekkene har jeg bare kopiert slik det var gjort i dialogmotearbeidsgiver fordi jeg synes det var vanskelig å lese og var redd for å ødelegge noe. Men jeg fikser!

Reply:
Skal se litt mer ordentlig på det etter levering i bhg :)